controllers/mysqlcluster_controller.go

```diff
@@ -456,6 +510,7 @@ func (r *MySQLClusterReconciler) createPasswordSecretForInit(ctx context.Context
 			return err
 		}
 	}
+
```

Comment:
The other passwords (operator, replica, donorPass) can be generated here.  I think It'll be cleaner.

Reply:
Thanks!
I fixed in this commit: https://github.com/cybozu-go/moco/pull/185/commits/6425fbf9621ef0ba901ef19c1b1ffca591e2812b